ui/src/features/Dashboard/components/Nav/components/WorkspaceNavigation.tsx

```diff
@@ -9,22 +9,23 @@ import {
   DropdownMenuItem,
   DropdownMenuTrigger,
 } from "@flow/components";
+import { useGetWorkspaceQuery } from "@flow/lib/gql";
 import { cn } from "@flow/lib/utils";
-import { workspaces } from "@flow/mock_data/workspaceData";
-import { useCurrentProject, useCurrentWorkspace } from "@flow/stores";
+import { useCurrentWorkspace } from "@flow/stores";
 import { Workspace } from "@flow/types";
 
 const WorkspaceNavigation: React.FC = () => {
-  const [currentWorkspace, setCurrentWorkspace] = useCurrentWorkspace();
-  const [, setCurrentProject] = useCurrentProject();
-  const navigate = useNavigate({ from: "/workspace/$workspaceId" });
+  const [currentWorkspace, _] = useCurrentWorkspace();
```

Comment:
You can just write `const [currentWorkspace] ....`. The others not being used is inferred.